### PR TITLE
Issue #287

### PR DESCRIPTION
**Pull request by kirjaamo**
Added links to disabling SELinux and AppArmor.

### CODE REVIEW

File: galeracluster/source/documentation/install-mysql.rst [hunk 1]
```diff
@@ -210,6 +212,8 @@ After you've created, modified, and saved this repository file, you'll be able t
 
 There are two packages involved in the installation of Galera Cluster for MySQL: the MySQL database server, but one that has been built to include the :term:`wsrep API`; and the :term:`Galera Replication Plugin`. The ``yum`` repositories include Galera Arbitrator with the Galera Replication Plugin, but for Debian-based distributions using ``apt-get`` you'll need to include add it to your installation instruction.
 
+.. note:: If SELinux (Security-Enhanced Linux) is enabled on the servers, disable it. See :ref:`Disabling SELinux for mysqld <disable-selinux>`. Also, if AppArmor is enabled on the servers, disable it. See :ref:`Disabling AppArmor <disable-apparmor>`.
```
Comment (byte) [inline galeracluster/source/documentation/install-mysql.rst:215]: mention that this is "optional", and there are ways to enable the context files

File: galeracluster/source/documentation/install-mysql.rst [hunk 2]
```diff
@@ -224,25 +228,13 @@ For Galera Cluster 8.4:
 
    apt-get install galera-4 galera-arbitrator-4 mysql-wsrep-8.4
```
Comment (byte) [inline galeracluster/source/documentation/install-mysql.rst:229]: please do not remove 8.. for 8.0 the 8.0 insturctions must exist. for 8.4 this is ok

File: galeracluster/source/documentation/install-mysql.rst [hunk 3]
```diff
@@ -224,25 +228,13 @@ For Galera Cluster 8.4:
...
 .. code-block:: console
 
-   yum install galera-3 mysql-wsrep-8.4
-
-Please note that on CentOS 6 and 7, this command may generate a transaction check error. For more information on that error and how to resolve it, see the section below on :ref:`MySQL Shared Compatibility Libraries <centos-mysql-shared-compt>`.
-
-If you want to install version 4 of Galera Cluster using ``yum``, you would instead execute the following from the command-line:
-
-.. code-block:: console
+   yum install galera-4 mysql-wsrep-8.4
```
Comment (byte) [inline galeracluster/source/documentation/install-mysql.rst:235]: please do not remove 8.. for 8.0 the 8.0 insturctions must exist. for 8.4 this is ok

File: galeracluster/source/documentation/install-mysql.rst [hunk 4]
```diff
@@ -224,25 +228,13 @@ For Galera Cluster 8.4:
...
 
-   yum install galera-4 mysql-wsrep-8.0
+.. note:: On CentOS 6 and 7, this command may generate a transaction check error. For more information on that error and how to resolve it, see the section below on :ref:`MySQL Shared Compatibility Libraries <centos-mysql-shared-compt>`.
```
Comment (byte) [inline galeracluster/source/documentation/install-mysql.rst:237]: we do not build for centos 6 any longer.

**Review by byte — CHANGES_REQUESTED** [review]
changes please

File: galeracluster/source/documentation/install-mysql.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/install-mysql.rst]: Fixed

File: galeracluster/source/documentation/install-mysql.rst (hunk 2 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/install-mysql.rst:229]: Fixed

File: galeracluster/source/documentation/install-mysql.rst (hunk 3 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/install-mysql.rst:235]: Fixed

File: galeracluster/source/documentation/install-mysql.rst (hunk 4 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/install-mysql.rst]: Fixed

**Review by byte — APPROVED** [review]
ok to push

**Pull request merged**